modules/test/playwright/tests/journal-web/journalArticle.spec.ts

```diff
@@ -111,6 +111,30 @@ baseTest(
 	}
 );
 
+baseTest(
+	'Check error message on invalid friendly URL',
+	{
+		tag: '@LPD-38754',
+	},
+	async ({journalEditArticlePage, site}) => {
+		await journalEditArticlePage.journalPage.goto(site.friendlyUrlPath);
```

Comment:
No need of nesting here, just use 
```
await journalPage.goto(site.friendlyUrlPath);
await journalPage.goToCreateArticle();
```
 isn't it?

Reply:
do not know 🤷🏻‍♀️

Reply:
No, `journalPage` is not available on the test file so...

await journalEditArticlePage.goto(site.friendlyUrlPath); is ok?

Reply:
That is much better `await journalEditArticlePage.goto(site.friendlyUrlPath);`
because it includes the call to `.goToCreateArticle();` 👍